ie_serving/server/start.py

```diff
@@ -39,7 +42,7 @@ def initialize_tf():
     pass
 
 
-def serve(models, max_workers: int=1, port: int=9000):
+def serve(models, rest_thread=None, max_workers: int=1, port: int=9000):

```

Comment:
I don't see usage of `rest_thread` parameter in this function. What is the purpose of it?